library/network/src/modules/SuSEFirewall.rb

```diff
@@ -655,6 +655,29 @@ def morph_to(backend_sym)
       Yast.const_set('SuSEFirewall', self.class.superclass.create(backend_sym))
     end
 
+    # Local function for increasing the verbosity level.
```

Comment:
You are probably right. I think I made a mistake here and though that there were modules tweaking the verbosity of the backend.